rom/dev/tests/rom_integration_tests/test_wdt_activation_and_stoppage.rs

```diff
@@ -107,4 +107,15 @@ fn test_rom_wdt_timeout() {
     .unwrap();
 
     hw.step_until(|m| m.soc_ifc().cptra_fw_error_fatal().read() == WDT_EXPIRED);
+
+    // no mcause if wdt times out
+    assert_eq!(hw.soc_ifc().cptra_fw_extended_error_info().at(0).read(), 0);
+    // no mscause if wdt times out
+    assert_eq!(hw.soc_ifc().cptra_fw_extended_error_info().at(1).read(), 0);
+    // mepc is a memory address so won't be 0
+    assert_ne!(hw.soc_ifc().cptra_fw_extended_error_info().at(2).read(), 0);
```

Comment:
Can you print out the address somewhere so folks can manually verify it looks sane?

Reply:
I added some code printing it out, but if the watchdog timer expires, won't the nmi handler print the error codes to UART already?